File: C++/Rectangle.cpp
#include <iostream>

using namespace std;

int main() {
    // Variable for Rectangle 1 inputs
    double Width1, Length1;

    // Variable for Rectangle 2 inputs
    double Width2, Length2;

    // Getting inputs //
    // Rectangle 1 Inputs
    cout << "\n=========== Rectangle 1 Inputs ============" << endl;
    cout << "Enter Rectangle 1 Width: ";
    cin >> Width1;

    cout << "Enter Rectangle 1 Height: ";
    cin >> Length1;

    double ReactangleArea1 = (Width1 * Length1); // Calculate
    cout << "Rectangle 1 Area: " << ReactangleArea1 << endl;

    // Rectangle 2 Inputs
    cout << "\n=========== Rectangle 2 Inputs ============" << endl;
    cout << "Enter Rectangle 2 Width: ";
    cin >> Width2;

    cout << "Enter Rectangle 2 Height: ";
    cin >> Length2;

    // Calculations
    double ReactangleArea2 = (Width2 * Length2);
    cout << "Rectangle 2 Area: " << ReactangleArea2 << endl;

    cout << "\n==================" << endl;

    // Checking and Outputing!!
    if (ReactangleArea1 > ReactangleArea2) {
        cout << "Rectangle 1 is greater that rectangle 2";
    } else if (ReactangleArea2 > ReactangleArea1) {
        cout << "Rectangle 2 is greater than Rectangle 1";
    } else {
        cout << "Both Rectangles are the Same Size";
    }

}

File: Java/Rectangle.java
import java.util.*;

public class Rectangle {

    public static void main(String[] args) {
        
        // Stores the value for Rectangle 1
        double width1;
        double length1;
        double Rectangle_Area_1;

        // Stores the value for Rectangle 2
        double width2;
        double length2;
        double Rectangle_Area_2;


        try ( Scanner s = new Scanner(System.in) ) {
            // Inputs for Rectangle 1
            System.out.println("============\nRectangle 1\n============\n");

            System.out.print("Input Width for Rectangle 1: ");
            width1 = s.nextDouble();
            System.out.print("Input Length for Rectangle 1: ");
            length1 = s.nextDouble();

            Rectangle_Area_1 = (length1 * width1);
            System.out.println("Rectangle 1 Area: " + Rectangle_Area_1);

            // Inputs for Rectangle 2
            System.out.println("\n============\nRectangle 2\n============\n");

            System.out.print("Input Width for Rectangle 2: ");
            width2 = s.nextDouble();
            System.out.print("Input Length for Rectangle 2: ");
            length2 = s.nextDouble();

            Rectangle_Area_2 = (length2 * width2);
            System.out.println("Rectangle 2 Area: " + Rectangle_Area_2);
            s.close();
        }

        System.out.println("\n====================\n");

        if (Rectangle_Area_1 == Rectangle_Area_2) {
            System.out.println("Both Rectangle Areas are the Same!");
        } else if (Rectangle_Area_1 > Rectangle_Area_2) {
            System.out.println("Rectangle 1 Area is greater than Rectangle Area 2");
        } else if (Rectangle_Area_1 < Rectangle_Area_2) {
            System.out.println("Rectangle 2 Area is greater than Rectangle Area 1");
        }

    }

}